File: Easy/Java/728. Self Dividing Numbers.java
class Solution {
    public List<Integer> selfDividingNumbers(int left, int right) {
        List<Integer> list = new ArrayList();

        for(int i = left; i <= right; i++){
            int number = i;
            while(0 < number){
                if((number % 10) != 0 && i % (number % 10) == 0)
                    number /= 10;
                else
                    number = -1;
            }
            if(number != -1)
                list.add(i);
        }

        return list;
    }
}

File: Easy/Python/728. Self Dividing Numbers.py
class Solution(object):
    def selfDividingNumbers(self, left, right):
        myList = []

        for i in range(left, right + 1):
            number = i
            while(0 < number):
                if(number % 10) != 0 and i % (number % 10) == 0:
                    number = number / 10
                else:
                    number = -1
            if number != -1:
                myList.append(i)

        return myList
